### PR TITLE
[type:fix] Make the default value of ShenyuDubboService annotation equal to that of DubboService annotation

### LINKED ISSUE #5815: [BUG] ShenyuDubboService annotation some default value not equals DubboService annotation
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

https://github.com/apache/shenyu/blob/master/shenyu-client/shenyu-client-dubbo/shenyu-client-apache-dubbo/src/main/java/org/apache/shenyu/client/apache/dubbo/annotation/ShenyuDubboService.java

https://github.com/apache/dubbo/blob/3.3/dubbo-common/src/main/java/org/apache/dubbo/config/annotation/DubboService.java

You can see that some default values for the two annotations are different. For example, timeout, with a default value of -1 in DubboService and 0 in ShenyuDubboService, results in some metadata being registered with a timeout of 0, while others have a timeout of 3000.

iShot_2024-11-27_14 44 24
iShot_2024-11-27_14 44 42

### Expected Behavior

_No response_

### Steps To Reproduce

_No response_

### Environment

```markdown
ShenYu version(s):
```

### Debug logs

_No response_

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by eye-gu**
close #5815 

Make sure that:

- [x] You have read the [contribution guidelines](https://shenyu.apache.org/community/contributor-guide).
- [x] You submit test cases (unit or integration tests) that back your changes.
- [x] Your local test passed `./mvnw clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

File: shenyu-client/shenyu-client-dubbo/shenyu-client-apache-dubbo/src/main/java/org/apache/shenyu/client/apache/dubbo/annotation/ShenyuDubboService.java [hunk 1]
```diff
@@ -290,15 +287,15 @@
...
+    int retries() default -1;
 
     /**
      * Alias for {@link DubboService#loadbalance()} .
      *
      * @return the string
      */
     @AliasFor(annotation = DubboService.class)
-    String loadbalance() default LoadbalanceRules.RANDOM;
+    String loadbalance() default LoadbalanceRules.EMPTY;
```
Comment (moremind) [inline shenyu-client/shenyu-client-dubbo/shenyu-client-apache-dubbo/src/main/java/org/apache/shenyu/client/apache/dubbo/annotation/ShenyuDubboService.java:298]: why empty

**Review by moremind — CHANGES_REQUESTED** [review]

File: shenyu-client/shenyu-client-dubbo/shenyu-client-apache-dubbo/src/main/java/org/apache/shenyu/client/apache/dubbo/annotation/ShenyuDubboService.java (hunk 1 above)
Comment (eye-gu) [reply shenyu-client/shenyu-client-dubbo/shenyu-client-apache-dubbo/src/main/java/org/apache/shenyu/client/apache/dubbo/annotation/ShenyuDubboService.java:298]: This is the default value for DubboService annotation. Empty and random are different. If the configuration file specifies load balancing is RoundRobin, then the final result for Empty is RoundRobin.

File: shenyu-client/shenyu-client-dubbo/shenyu-client-apache-dubbo/src/main/java/org/apache/shenyu/client/apache/dubbo/annotation/ShenyuDubboService.java [hunk 2]
```diff
@@ -202,7 +199,7 @@
      * @return the int
      */
     @AliasFor(annotation = DubboService.class)
-    int delay() default 0;
+    int delay() default -1;
```
[the remark this answers is not in the document]
Comment (eye-gu) [reply shenyu-client/shenyu-client-dubbo/shenyu-client-apache-dubbo/src/main/java/org/apache/shenyu/client/apache/dubbo/annotation/ShenyuDubboService.java:202]: https://github.com/apache/dubbo/blob/3.3/dubbo-config/dubbo-config-api/src/main/java/org/apache/dubbo/config/ServiceConfig.java

You can see line 337.

[the remark this answers is not in the document]
Comment (eye-gu) [reply shenyu-client/shenyu-client-dubbo/shenyu-client-apache-dubbo/src/main/java/org/apache/shenyu/client/apache/dubbo/annotation/ShenyuDubboService.java:202]: Keeping the default values consistent can make the behavior of two annotations consistent Even if there is currently no difference between the two, they are still compatible with Dubbo's future modifications

**Review by yu199195 — APPROVED** [review]

File: shenyu-client/shenyu-client-dubbo/shenyu-client-apache-dubbo/src/main/java/org/apache/shenyu/client/apache/dubbo/annotation/ShenyuDubboService.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (yu199195) [reply shenyu-client/shenyu-client-dubbo/shenyu-client-apache-dubbo/src/main/java/org/apache/shenyu/client/apache/dubbo/annotation/ShenyuDubboService.java:202]: Yes， good job.

**Review by moremind — APPROVED** [review]

**Pull request merged**